vicephp/Virtue-JWT/src/JWT/VerificationFailed.php

```diff
@@ -4,6 +4,14 @@
 
 class VerificationFailed extends \RuntimeException
 {
+    public const ON_TYPE = 1;
+    public const ON_CLAIM = 2;
+    public const ON_AUDIENCE = 3;
+    public const ON_ISSUER = 4;
+    public const ON_SUBJECT = 5;
+    public const ON_ALGORITHM = 6;
+    public const ON_TIME = 7;
```

Comment:
One more code we need is `ON_SIGNATURE`, that can be sort of default for the signature validators